src/fermilib/utils/_plane_wave_hamiltonian.py

```diff
@@ -162,9 +163,7 @@ def plane_wave_hamiltonian(n_dimensions, grid_length, length_scale, geometry,
         if item[0] not in periodic_hash_table:
             raise ValueError("Invalid nuclear element.")
 
-    jellium_op = jellium_model(n_dimensions,
-                               grid_length,
-                               length_scale,
+    jellium_op = jellium_model(Grid(n_dimensions, grid_length, length_scale),
```

Comment:
Probably the function plane_wave_hamiltonian() could just take "grid" as an argument, and pass the "grid" to plan_wave_u_operator() and dual_basis_u_operator() below.